store/postgres/secret_repository.go

```diff
@@ -155,27 +185,29 @@ func (repo *secretRepository) GetByID(ctx context.Context, id uuid.UUID) (models
 	return r.ToSpec(repo.hash)
 }
 
-func (repo *secretRepository) GetAll(ctx context.Context) ([]models.ProjectSecretItem, error) {
-	var specs []models.ProjectSecretItem
+func (repo *secretRepository) GetAll(ctx context.Context) ([]models.SecretItemInfo, error) {
+	var secretItems []models.SecretItemInfo
 	var resources []Secret
-	if err := repo.db.WithContext(ctx).Find(&resources).Error; err != nil {
-		return specs, err
+	if err := repo.db.WithContext(ctx).Preload("Namespace").
+		Joins("LEFT JOIN namespace ON secret.namespace_id = namespace.id").
+		Where("secret.project_id = ? and secret.type = 'user'", repo.project.ID).Find(&resources).Error; err != nil {
```

Comment:
we should use enum here for secret type instead of hardcoded string `user`

Reply:
Sure, will use the enum